src/contexts/DocumentListContext.jsx

```diff
@@ -31,7 +31,7 @@ export const DocumentListContextProvider = ({ children }) => {
   const [loadingDocuments, setLoadingDocuments] = useState(true);
   const { podUrl } = useContext(SignedInUserContext);
   const [client, setClient] = useState(null);
-  const { session } = useSession();
+  const { session } = useContext(SessionContext);
```

Comment:
I've noticed the switch with `SessionContext` for the `session` object. We can technically phase out the custom hook in favor `useContext(SessionContext)` in future changes, correct? Since `useSession` simply return everything that's already declared in `SessionContext`.

Reply:
The main difference in this PR between `useContext(SessionContext)` and `useSession` is that `useSession` filters and modifies some of the data from `sessionContext`. E.G. `SessionContext` exposes the full solid profile document, however I modified `useSession` to filter that down to just a pod Url so that each component doesn't need to compute the pod url to use.